feature/system/gnmi/cliorigin/mixed_oc_cli_origin_support_test/mixed_oc_cli_origin_support_test.go

```diff
@@ -46,9 +46,10 @@ func buildOCUpdate(path *gpb.Path, value string) *gpb.Update {
 	if len(path.GetElem()) == 0 || path.GetElem()[0].GetName() != "meta" {
 		path.Origin = "openconfig"
 	}
+	json_value := fmt.Sprintf("\"%s\"", value)
```

Comment:
Our tests use camel case, can you change this to something like `jsonVal`?

Reply:
The ygot way to convert to Json_ietf would be

`jsonVal, err := ygot.Marshal7951(ygot.String(value))`